core/main/src/state/openrpc_state.rs

```diff
@@ -442,6 +446,28 @@ impl OpenRpcState {
             .unwrap()
             .extend(provider_relation_sets)
     }
+
```

Comment:
Why not cache in the validator (eventually a crate) so it's universal?

Reply:
Validator Crate uses serde to build the object it cannot have a thread safe box in it
#[derive(Debug, Serialize, Deserialize, Clone)]
pub struct FireboltOpenRpc {